src/Umbraco.Infrastructure/Routing/RedirectTrackingHandler.cs

```diff
@@ -99,25 +99,32 @@ private void StoreOldRoute(IContent entity, OldRoutesDictionary oldRoutes)
             {
                 return;
             }
-            var contentCache = publishedSnapshot.Content;
-            var entityContent = contentCache?.GetById(entity.Id);
+
+            IPublishedContentCache contentCache = publishedSnapshot.Content;
+            IPublishedContent entityContent = contentCache?.GetById(entity.Id);
             if (entityContent == null)
```

Comment:
might aswell change this while we're here
```suggestion
            if (entityContent is null)
```

Reply:
Good point 👍